autocompleter/base.py

```diff
@@ -604,17 +602,19 @@ def suggest(self, term, facets=[]):
                     except KeyError:
                         continue
                 # We want to calculate the intersection of all the intermediate facet sets created so far
-                # along with the base term result set. So we need to use a new unique name for the
-                # intermediate result set and append the base_term_result_key to the list of
+                # along with the base result set. So we need to use a new unique name for the
+                # intermediate result set and append the base_result_key to the list of
                 # facet_result_keys.
-                base_result_key = RESULT_SET_BASE_NAME % str(uuid.uuid4())
-                facet_result_keys.append(base_term_result_key)
-                pipe.zinterstore(base_result_key, facet_result_keys, aggregate='MIN')
+                faceted_base_result_key = RESULT_SET_BASE_NAME % str(uuid.uuid4())
```

Comment:
since the only thing we do with this is  store then query, seems same to leave this out of the provider loop  ... this also applies for facet_exact_match_key
is there a reason there generation inside the provider loop that's i'm missing....

these seem no different in usage than base_result_key